integration-cli/docker_cli_volume_test.go

```diff
@@ -45,8 +46,20 @@ func (s *DockerSuite) TestVolumeCLIInspect(c *check.C) {
 	c.Assert(strings.TrimSpace(out), check.Equals, name)
 
 	dockerCmd(c, "volume", "create", "test")
+	now := time.Now()
 	out, _ = dockerCmd(c, "volume", "inspect", "--format={{ .Name }}", "test")
 	c.Assert(strings.TrimSpace(out), check.Equals, "test")
+
+	// using the same volume to inspect and test that the CreatedAt field contains the time the volume was created
```

Comment:
@mtesselH this should go into `docker_api_volumes_test.go` (probably in `TestVolumeCLIInspect` https://github.com/moby/moby/blob/master/integration-cli/docker_api_volumes_test.go#L68)

Probably something like :

```diff
diff --git a/integration-cli/docker_api_volumes_test.go b/integration-cli/docker_api_volumes_test.go
index 3cc03dfb3..d35c09df4 100644
--- a/integration-cli/docker_api_volumes_test.go
+++ b/integration-cli/docker_api_volumes_test.go
@@ -4,6 +4,7 @@ import (
        "encoding/json"
        "net/http"
        "path/filepath"
+       "time"

        "github.com/docker/docker/api/types"
        volumetypes "github.com/docker/docker/api/types/volume"
@@ -72,6 +73,7 @@ func (s *DockerSuite) TestVolumesAPIInspect(c *check.C) {
        status, b, err := request.SockRequest("POST", "/volumes/create", config, daemonHost())
        c.Assert(err, check.IsNil)
        c.Assert(status, check.Equals, http.StatusCreated, check.Commentf(string(b)))
+       now := time.Now()

        status, b, err = request.SockRequest("GET", "/volumes", nil, daemonHost())
        c.Assert(err, checker.IsNil)
@@ -87,4 +89,8 @@ func (s *DockerSuite) TestVolumesAPIInspect(c *check.C) {
        c.Assert(status, checker.Equals, http.StatusOK, check.Commentf(string(b)))
        c.Assert(json.Unmarshal(b, &vol), checker.IsNil)
        c.Assert(vol.Name, checker.Equals, config.Name)
+       //Check that the time returned is valid and equal to the time the volume was created at. Truncate a minute from both times to avoid false postive fails
+       now = now.Truncate(time.Minute)
+       createdAt := vol.CreatedAt.Truncate(time.Minute)
+       c.Assert(now, checker.Equals, createdAt, check.Commentf("Time Volume is CreatedAt not equal to current time"))
 }
```